src/bridge/interface.cairo

```diff
@@ -0,0 +1,94 @@
+use starknet::ContractAddress;
+
+#[derive(Serde, Drop, starknet::Store, PartialEq)]
+pub enum TokenStatus {
+    #[default]
+    Unknown,
+    Pending,
+    Active,
+    Blocked
+}
+
+#[derive(Serde, Drop, starknet::Store)]
+pub struct TokenSettings {
+    pub token_status: TokenStatus,
+    pub deployment_message_hash: felt252,
+    pub pending_deployment_expiration: u64,
+    pub max_total_balance: u256,
+    pub withdrawal_limit_applied: bool
+}
```

Comment:
is it normal to have enum and structs in an interface? i would expect these to be in a types.cairo file normally

Reply:
Removed, added to types.cairo